.kokoro/release/publish_javadoc.sh

```diff
@@ -53,3 +58,19 @@ python3 -m docuploader upload . \
   --staging-bucket ${STAGING_BUCKET}
 
 popd
+
+# V2
+mvn clean site -B -Ddevsite.template="$(dirname `pwd`)/doc-templates/java/"
```

Comment:
Can we make this relative to this file rather than where it's being executed from? In the future, it's possible we refactor how these scripts are executed and it'll be one fewer thing to think about.

Reply:
Well, I tried that originally, but the variable is passed in w/o expansion, so when the subproject is built, it's the wrong one.